### PR TITLE
Resizable log view #3833

### PR DESCRIPTION
**Pull request by idotj**
Replaced the `<div>` containing the complete log by a `<textarea readonly>`
This will allow the user to resize the width and height if clicks and drag at the bottom right corner. Also vertical and horizontal scroll appears when needed (native behavior for `<textarea>`)

I also pushed the copy log link to the right side and added the label/text.

### CODE REVIEW

**Comment by warwickmm** [comment]
Thanks @idotj!  When I click "Copy log", the content is copied to the clipboard, but the text in the textarea is also selected.  This might give users the impression that this merely selects the text, and they still have to manually perform the copy.  Is it possible for the text to be copied to the clipboard without visually indicating the selection?

image

**Comment by warwickmm** [comment]
Sorry, I just noticed that's the existing behavior as well.  We can leave that for a separate pull request in the future.

File: Localizations/webroot/localization_webroot-bn.po [hunk 1]
```diff
@@ -449,6 +449,10 @@ msgstr ""
 msgid "Complete log"
 msgstr ""
 
+#: templates/backup-result/box.html:40
+msgid "Copy log"
+msgstr ""
+
```
Comment (warwickmm) [inline Localizations/webroot/localization_webroot-bn.po:455]: I believe the `.po` and `.pot` files are automatically created when we update the translations prior to a release, so we do not need to modify them here.

Comment (idotj) [reply Localizations/webroot/localization_webroot-bn.po]: Ok, I removed all the changes for .po and .pot files, should be fine.

**Comment by warwickmm** [comment]
Thanks @idotj!.

**Pull request merged**